cultivation/scripts/task_management/session_timer.py

```
@@ -0,0 +1,249 @@
+    if log_dir and not os.path.exists(log_dir):
+        os.makedirs(log_dir, exist_ok=True)
+
+    # --- RECOVERY LOGIC ---
+    unclosed = parse_log_for_unclosed(log_file=LOG_FILE)
+    if unclosed:
+        print("\n=== Session Recovery Needed ===")
+        for idx, (task, start, note) in enumerate(unclosed, 1):
+            print(f"[{idx}] Task: {task}, Started: {start}, Note: {note}")
+        for task, start, note in unclosed:
+            print(f"Recovering session for Task {task} started at {start}...")
+            end_time_str = prompt(f"Enter end time for Task {task} (YYYY-MM-DDTHH:MM, default=now)", default=datetime.now().strftime("%Y-%m-%dT%H:%M"))
```

Pylint (3.3.7)

[convention] Line too long (151/100)

(C0301)